micromasters/urls.py

```diff
@@ -27,8 +28,8 @@
     url(r'^api/v0/dashboard/$', UserDashboard.as_view(), name='dashboard_api'),
     url(r'^api/v0/search/(?P<elastic_url>.*)', ElasticProxyView.as_view(), name='search_api'),
     url(r'^api/v0/checkout/$', CheckoutView.as_view(), name='checkout'),
+    url(r'^api/v0/mail$', MailView.as_view(), name='mail_api'),
```

Comment:
If you add the slash here and to the JS at the same time I don't think you should get this error. All of our other URLs have a trailing slash so it would be nice to be consistent